src/DenseMatrixinverter.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2017-2018  Waterloo Quantitative Consulting Group, Inc.
+// See COPYING and LICENSE files at project root for more details.
+
+#include "DenseMatrixInverter.hpp"
+#include "DenseMatrixHelpers.hpp"
+#include "Types.hpp"
+#include <string>
+#include <stdexcept>
```

Comment:
If you use std::stringstream then you should add:  #include<sstream>

Reply:
I think iomanip has this as well...